docs/source/initialsetup/configuringmame.rst

```diff
@@ -0,0 +1,78 @@
+Configuring MAME
+----------------
+
+.. contents:: :local:
+
+Getting Started: A Quick Preface
+--------------------------------
+
+Once you have MAME installed, the next step is to configure it. There are
+several ways to do this, and each will be covered in turn.
+
+If you are on Windows, the MAME executable will be called ``mame64.exe`` or
+``mame.exe`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+If you are on Linux or MacOS, the MAME executable will be called ``mame64`` or
+``mame`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+
+Initial Setup: Creating mame.ini From Command Line on Windows
+-------------------------------------------------------------
+
+First, you will need to *cd* to the directory where you installed MAME into.
+If, for instance, you have MAME installed in ``c:\users\public\mame`` you will
+need to type ``cd c:\users\public\mame`` into the command prompt.
```

Comment:
Once again, case-sensitive filesystems do exist (despite being rare on Windows).  The canonical form is “C:\Users\Public” – better safe than sorry.

Reply:
Adjusted.